gsoc/background.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title:  "Background on GSoC: start here!"
+show_main: false 
+---
+
+# Background: Start Here!
+
+### What is GSoC? 
+
+Google Summer of Code is a remote summer coding internship program funded by 
+Google. Google distributes funds to open software development organizations to 
```

Comment:
I'm not sure whether `awards` would be better than `distributes`. I get the impression with distributes that they do to everyone.